docs/sources/flow/reference/components/loki.write.md

```diff
@@ -0,0 +1,154 @@
+`batch_wait`          | `bool`     | Maximum amount of time to wait before sending a batch. | `"1s"` | no
+`batch_size`          | `string`   | Maximum batch size of logs to accumulate before sending. | `"1MiB"` | no
+`remote_timeout`      | `duration` | Timeout for requests made to the URL. | `"10s"`        | no
+`tenant_id`           | `string`   | The tenant ID used by default to push logs. |  | no
+`min_backoff_period`  | `duration` | Initial backoff time between retries. | `"500ms"`      | no
+`max_backoff_period`  | `duration` | Maximum backoff time between retries. | `"5m"`         | no
+`max_backoff_retries` | `int`      | Maximum number of retries.            | 10             | no
+
+
+If no `tenand_id` is provided, the component assumes that the Loki instance at
+`endpoint` is running in single-tenant mode and no X-Scope-OrgID header is
+sent.
+
+When multiple `endpoint` blocks are provided, the `loki.write` component 
+creates a client for each. Received log entries are fanned-out to these clients
+in succession. That means that if one client is bottlenecked, it may impact
+the rest of them, too.
+
+Endpoints can be named for easier identification in debug metrics using the
+`name` argument. If the `name` argument isn't provided, a name is generated
+based on a hash of the endpoint settings.
+
+### http_client_config block
+
+The `http_client_config` configures settings used to connect to the Loki API.
+
+{{< docs/shared lookup="flow/reference/components/http-client-config-block.md" source="agent" >}}
+
+### basic_auth block
+
+{{< docs/shared lookup="flow/reference/components/basic-auth-block.md" source="agent" >}}
+
+### authorization block
+
+{{< docs/shared lookup="flow/reference/components/authorization-block.md" source="agent" >}}
+
+### oauth2 block
+
+{{< docs/shared lookup="flow/reference/components/oauth2-block.md" source="agent" >}}
+
+### tls_config block
+
+{{< docs/shared lookup="flow/reference/components/tls-config-block.md" source="agent" >}}
+
+## Exported fields
+
+The following fields are exported and can be referenced by other components:
+
+Name | Type | Description
+---- | ---- | -----------
+`receiver` | `LogsReceiver` | A value that other components can use to send log entries to.
+
+## Component health
+
+`loki.write` is only reported as unhealthy if given an invalid
+configuration.
+
+## Debug information
+
+`loki.write` does not expose any component-specific debug
+information.
+
+## Debug metrics
+* `loki_write_encoded_bytes_total` (counter): Number of bytes encoded and ready to send.
+* `loki_write_sent_bytes_total` (counter): Number of bytes sent.
+* `loki_write_dropped_bytes_total` (counter): Number of bytes dropped because failed to be sent to the ingester after all retries.
+* `loki_write_sent_entries_total` (counter): Number of log entries sent to the ingester.
+* `loki_write_dropped_entries_total` (counter): Number of log entries dropped because failed to be sent to the ingester after all retries.
```

Comment:
```suggestion
* `loki_write_dropped_entries_total` (counter): Number of log entries dropped because they failed to be sent to the ingester after all retries.
```